<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary" tests="14" time="874.996" ignored="0" failures="7" timestamp="14 Jun 2021 20:37:45 GMT" skipped="0" hostname="IN-L2504" errors="0">
  <testcase name="testGetNotificationsForNetworthSummary" time="8.730" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="9.485" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="65.339" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at org.testng.Assert.fail(Assert.java:94)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="66.020" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at org.testng.Assert.fail(Assert.java:94)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="101.892" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="99.168" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="59.674" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="68.218" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at org.testng.Assert.fail(Assert.java:94)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="66.303" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at org.testng.Assert.fail(Assert.java:94)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="135.870" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="8.092" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary"/>
  <testcase name="testGetNotificationsForNetworthSummary" time="66.500" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at org.testng.Assert.fail(Assert.java:94)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="53.525" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="InValid createdDate either format is not correct OR created date may be greater than current date">
      <![CDATA[java.lang.AssertionError: InValid createdDate either format is not correct OR created date may be greater than current date
	at org.testng.Assert.fail(Assert.java:94)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:315)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
  <testcase name="testGetNotificationsForNetworthSummary" time="66.180" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at org.testng.Assert.fail(Assert.java:94)
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary.testGetNotificationsForNetworthSummary(TestGetInsightsForNetworthSummary.java:295)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testGetNotificationsForNetworthSummary -->
</testsuite> <!-- com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForNetworthSummary -->
